ui/src/parts/lightbox.tsx

```diff
@@ -149,7 +139,15 @@ export const Lightbox = ({ images, defaultImageIdx }: LightboxProps) => {
       else if (ev.key === 'ArrowLeft') handleShowPrevImage()
     },
     handleCloseOnFreeSpaceClick = (ev: React.MouseEvent) => {
-      if ((ev.target as HTMLElement).parentElement === ev.currentTarget) lightboxB(null)
+      if (ev.target === ev.currentTarget) lightboxB(null)
+    },
+    handleCursorOnFreeSpaceHover = (ev: React.MouseEvent<HTMLElement>) => {
+      // Changes the cursor to 'pointer' when hovering over the free space.
+      if (ev.target === ev.currentTarget) ev.currentTarget.style.cursor = 'pointer'
+      else {
+        ev.currentTarget.style.cursor = ''
+        ev.currentTarget.style.cursor = getComputedStyle(ev.currentTarget).cursor
+      }
```

Comment:
This seems unnecessary. Couldn't it be achieved with a simple CSS rule?

Reply:
> Another option would be to specify the pointer cursor for body element and also cursors for every child element separately to override the parent's one.

Yes, that's what I meant. Such things should be done in JS only as last-resort solutions.